src/test/fixtures/sources.ts

```diff
@@ -22,35 +28,37 @@ export const mockSourcesByProvider = (provider) => {
         source_type_id: '3',
         uid: '73d5694c-7a28-417e-9fca-55840084f508',
       },
-    ];
-  }
+    ],
+  };
+  return mockSources[provider as keyof typeof mockSources];
 };
 
-export const mockUploadInfo = (sourceId) => {
-  if (sourceId === '666') {
-    return {
+export const mockUploadInfo = (
+  sourceId: string
+): GetSourceUploadInfoApiResponse => {
```

Comment:
There is some debate on whether inferred or explicit return types are the best but right now my idea is to write as *little* Typescript as possible, and I'm fine with inferred returns. I think if we type the mockInfo object below we don't need to specify the return type. Feel free to disagree, the most important thing for me is that we agree on a consistent style.

Reply:
Got it. I think in this case I would also be okay with inferred return as there's very little going on with the data along the way. On the other hand I'm wondering whether explicit returns could help in cases where there's significant transformation of the data. It would make the returns more rigid, but could also safeguard against errors. What do you think?

Do you see some possible problems with explicit returns? 
I can't really see any with the inferred returns other than more wiggle room for us but also maybe some :bug:?

Reply:
As discussed offline, let's go with explicit returns. :+1: